### DISCUSSION TITLE
June Personal Web - Golang Dependency Injection Using Wire

### DISCUSSION THREAD

**Discussion by giscus** [ClavinJune/ClavinJune.github.io#10] [General]
# June Personal Web - Golang Dependency Injection Using Wire

Implementing dependency injection in Golang using Google Wire

https://clavinjune.dev/en/blogs/golang-dependency-injection-using-wire/

**Comment by icodeerror** [comment]
Cool! Any chance to have this repo on Github?

**Reply by clavinjune** [reply]
Hi @icodeerror, thank you for waiting, here I created the repo for this blog post https://github.com/anon-org/golang-dependency-injection-using-wire
